tyk-pro/scripts/bootstrap.sh

```diff
@@ -22,6 +22,31 @@ syntax_message(){
   exit 1
 }
 
+check_for_existing_organisations(){
+    EXISTING_ORGS=$(CURL_CA_BUNDLE=$CURL_CA_BUNDLE_TYK curl --silent --header "admin-auth: $TYK_ADMIN_SECRET" --header "Content-Type:application/json"  $DASHBOARD_HOSTNAME/admin/organisations 2>&1)
+    OWNER_NAMES=$(echo "$EXISTING_ORGS" | jq -r '.organisations | .[] | .owner_name | @base64')
```

Comment:
Do we need to encode organisation name? Because at line 33, org_name is decoded again.

```suggestion
    OWNER_NAMES=$(echo "$EXISTING_ORGS" | jq -r '.organisations | .[] | .owner_name')
```

Reply:
Assume you have these two orgs: "Default org1" "Default org2"
If we wouldn't encode and decode it the space between the words "default" and "orgX" would cause our for loop to run 4 times instead of 2 times.

Reply:
🦸 makes sense. thanks!

Reply:
@andrei-tyk If you wrap whole expression inside [], it will return back an array.
So it will look like `jq -r ['.organisations | .[] | .owner_name']`